app/models/peep.rb

```diff
@@ -0,0 +1,15 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+require 'dm-migrations'
+
+class Peep
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :author, String, required: true
+  property :user_name, String, required: true
```

Comment:
what happens if the user changes their name?  Shouldn't the peep be directly related to the user?